extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -515,29 +515,45 @@ private static void setHttpServerTiming(InsecureRequests insecureRequests, HttpS
             HttpServerOptions sslConfig,
             HttpServerOptions domainSocketOptions) {
         String serverListeningMessage = "Listening on: ";
+        String applicationLinkMessage = "";
         int socketCount = 0;
 
         if (httpServerOptions != null && !InsecureRequests.DISABLED.equals(insecureRequests)) {
             serverListeningMessage += String.format(
                     "http://%s:%s", httpServerOptions.getHost(), actualHttpPort);
             socketCount++;
+
+            if (isLocalhost(httpServerOptions.getHost())) {
+                applicationLinkMessage += String.format("http://127.0.0.1:%s", actualHttpPort);
+            }
         }
 
         if (sslConfig != null) {
             if (socketCount > 0) {
                 serverListeningMessage += " and ";
+                applicationLinkMessage += " and ";
             }
             serverListeningMessage += String.format("https://%s:%s", sslConfig.getHost(), actualHttpsPort);
             socketCount++;
+
+            if (isLocalhost(sslConfig.getHost())) {
+                applicationLinkMessage += String.format("https://127.0.0.1:%s", actualHttpsPort);
+            }
         }
 
         if (domainSocketOptions != null) {
             if (socketCount > 0) {
                 serverListeningMessage += " and ";
+                applicationLinkMessage += " and ";
             }
             serverListeningMessage += String.format("unix:%s", domainSocketOptions.getHost());
+
+            if (isLocalhost(domainSocketOptions.getHost())) {
+                applicationLinkMessage += "unix:127.0.0.1";
```

Comment:
This confuses me... we have unix domain socket support?  But even if we did, this doesn't seem to make sense...

Reply:
You are right. Removed it in the latest push.